blog/Blog-2016-03-11-Grunt-1.0.0-rc2-released.md

```diff
@@ -0,0 +1,27 @@
+The Grunt team is pleased to announce the **second release candidate** for `1.0.0`.
+
+This release is made possible by [@shama](https://github.com/shama), [@vladikoff](https://github.com/vladikoff), [@jkaussieskater](https://github.com/jkaussieskater), [@dmethvin](https://github.com/dmethvin).
```

Comment:
I'd like to also give credit to @AurelioDeRosa, given the significant recent contributions made to the Docs!